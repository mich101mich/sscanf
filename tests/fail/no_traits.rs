struct Test;

fn main() {
    sscanf::scanf!("hi", "{}", Test);
}
