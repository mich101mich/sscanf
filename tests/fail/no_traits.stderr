error[E0277]: the trait bound `Test: RegexRepresentation` is not satisfied
 --> $DIR/no_traits.rs:4:5
  |
4 |     sscanf::scanf!("hi", "{}", Test);
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `RegexRepresentation` is not implemented for `Test`
  |
  = note: required by `REGEX`
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
